.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish to PyPI.org
+on:
+  release:
+    types: [published]
+jobs:
+  pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: python3 -m pip install --upgrade build && python3 -m build # build the python package
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
It is recommended to use a SHA instead of a release number, could you please add here the latest one instead of `v1`?

Reply:
Please can you eleaborate, I'll make the required changes asap.

Reply:
They recommend using `pypa/gh-action-pypi-publish@5fb2f047e26679d7846a8370de1642ff160b9025` , with an explicit git commit SHA,  instead of a version number like `v1` that might change its meaning over time

Reply:
@pberkes Okay..So now should I use the `SHA you gave` or `SHA from commit history?`
#### Last 3 commits with SHA values
![image](https://user-images.githubusercontent.com/79012023/196236928-b333bbca-d04e-4192-b21a-88fb9f5360ac.png)

#### Please specify which SHA to use and I'll add that SHA value, thank you.

Reply:
the SHA value should be of the `pypa/gh-action-pypi-publish` repository, and the one I wrote in the comment above will work :)

Reply:
@pberkes Alright, I'll add the `SHA` you gave...